roles/system_settings/tasks/proxy.yml

```diff
@@ -18,8 +17,7 @@
     server: "{{ item.url | mandatory }}"
     port: "{{ item.port | mandatory }}"
     protocol: "{{ item.protocol | mandatory }}"
-    state: "{{ item.state | d(omit) }}"
+    state: "{{ 'set' if item.state | d('present') == 'present' else item.state | d(omit) }}"
```

Comment:
It doesnt look like the omit default can ever be reached so i think it should be removed